src/tests/functional/test_deploy.py

```diff
@@ -9,7 +9,7 @@
 pytestmark = pytest.mark.asyncio
 
 series = ['xenial', 'bionic']
-juju_repository = os.getenv('JUJU_REPOSITORY', '.').rstrip('/')
+juju_repository = os.getenv('juju_repository', '.').rstrip('/')
```

Comment:
What's this for?

Reply:
Apparently just for breaking functional testing.
```
:undo
:wq
```
Fixed